app/routes.js

```diff
@@ -1,7 +1,7 @@
-const signIn = require('./controllers/singIn');
+const signIn = require('./controllers/signIn'),
+  signUp = require('./controllers/signUp');
 
 exports.init = app => {
-  // app.get('/endpoint/get/path', [], controller.methodGET);
-  // app.put('/endpoint/put/path', [], controller.methodPUT);
-  app.post('/user', signIn.signUp);
+  app.post('/user', signUp.signUp);
+  app.post('/user', signIn.sesion);
```

Comment:
You have duplicated the route, one should be sign in I guess

Reply:
no, is not duplicated. 
I changed the Files name, so signUp now actually signs up, and the method is called signUp  and sign in  now signs in using the sesion method